zerver/openapi/zulip.yaml

```diff
@@ -7555,6 +7555,14 @@ paths:
           schema:
             type: boolean
           example: true
+        - name: display_organization_name
+          in: query
+          description: |
+            Whether display the organization name in left-sidebar.
+            **Changes**: New in Zulip 5.0 (feature level 114).
+          schema:
```

Comment:
Small typo here, you're missing 'to' after 'Whether'.

Also, adding an empty line between the setting description and the **Changes** note will make sure these updates are rendered correctly in the api documentation. Same note below for the other additions.

I find it helpful to always check any updates to this file in the api docs generated in the dev environment, which you can access by going to: localhost:9991/api/